.nuspec/Microsoft.Maui.Controls.SingleProject.targets

```diff
@@ -78,6 +80,10 @@
   <ItemGroup Condition=" '$(SingleProject)' == 'true' ">
     <ProjectCapability Include="MauiSingleProject" />
     <ProjectCapability Include="LaunchProfiles" />
+    <!-- If VS is older than Dev17 -->
+    <ProjectCapability Include="XamarinStaticLaunchProfiles" Condition=" '$(VisualStudioVersion)' != '' and '$(VisualStudioVersion)' &lt; '17.0' " />
+    <!-- Otherwise define LaunchProfilesGroupByPlatformFilters by default -->
+    <ProjectCapability Include="LaunchProfilesGroupByPlatformFilters" Condition=" '$(VisualStudioVersion)' == '' or '$(VisualStudioVersion)' &gt;= '17.0' " />
```

Comment:
The comment looks backwards? This first check is `&lt;` aka `<`?

Reply:
I think it's right? `XamarinStaticLaunchProfiles` in Dev16 and `LaunchProfilesGroupByPlatformFilters` in Dev17? @tondat is that correct?